src/applications/gi/components/EstimateYourBenefitsSummarySheet.jsx

```diff
@@ -1,96 +1,141 @@
 import React from 'react';
-import { CalculatorResultRow as ImportedCalculatorResultRow } from '../components/profile/EstimatedBenefits';
+import classNames from 'classnames';
 
-const CalculatorResultRow = ({ label, value, header, bold, visible }) => (
-  <ImportedCalculatorResultRow
-    id={`summary-sheet-${label}`}
-    label={label}
-    value={value}
-    header={header}
-    bold={bold}
-    visible={visible}
-  />
-);
+const CalculatorSheetResultRow = ({
+  id,
+  label,
+  value,
+  header,
+  visible,
+  bold,
+  boldLabel,
+  plainTextValue,
+}) =>
+  visible ? (
+    <div
+      id={`calculator-result-row-${id == null ? label : id}`}
+      className={classNames('row', 'calculator-result', { bold })}
```

Comment:
should there be a check like so that if both `bold` and `boldLabel` are used everything isn't bolded ?
```
const boldAll = !boldLabel && bold;
```

Reply:
I thought about adding in a check to prevent things like this, but wasn't entirely sure sure what the expected/desired result would be if both were present. I can add that in though.